proxy/src/session.rs

```diff
@@ -228,4 +230,23 @@ pub mod settings {
             Self::Emulator
         }
     }
+
+    /// `CoCo` config parameters subject to user preferences
+    #[derive(Debug, Deserialize, Serialize, PartialEq)]
+    pub struct CoCo {
```

Comment:
Not sure `CoCo` naming makes sense for me. What do we think of calling it `Seeds` instead? Or `Garden`? :smirk:

Reply:
No @FintanH that would be too cute and fun.

Reply:
No @FintanH, that would be too cute and fun.

Reply:
We need more puns tbqhwy 😅 but ya CoCo doesn't cut it for me. Still to general for me

Reply:
I think if the only coco-related configuration we're doing right now is an array of `seeds`, we could call it `Seeds`. Going forward though, aren't there going to be other coco/p2p settings?

Reply:
Let's call it `Seeds` then :) 

> Going forward though, aren't there going to be other coco/p2p settings?

Could be, but let's not pre-optimise. Renaming things is easier with a compiler to catch things for you and we're not some backwards-compat lib, so let's take our liberties and convey the current intent of this structure.

Reply:
Strongly disagree with this change. If we built up namespace to group and bucket configuration paremeters `seeds.seeds` is not communicating at all what is referred to. Instead we should be true to the fact, that these are the seeds used for the p2p code-collab protocol. There might be more settings related to that in the future and if we don't have an adequate alternative by way of names, I'd like to see this changed back to `CoCo` as this is the nomenclatue in the rest fo the codebase as well.

We can debate to adopt a different naming in a separate conversation and outside of this change-set.

Reply:
Well I'd say make it `pub struct Seed(Vec<String>)` in this case.

I'm not willing to die on a hill over naming :) Sorry for misleading you in this case @sarahscott

Reply:
kein problem 😄  it happens, happy to get this one merged. we can change the name to `p2p` or `peer` down the line if we get tired of `coco`